Dane/Intro/MainMenu/OUTRO/outro2/OUTRO_2.cnv

```diff
@@ -38,106 +48,358 @@ STATEK:PRELOAD=TRUE
 STATEK:RELEASE=TRUE
 STATEK:MONITORCOLLISION=FALSE
 STATEK:MONITORCOLLISIONALPHA=FALSE
-
+STATEK:ONINIT=THISPLAY
 
 ######################################
 ##############  TIMER  ###############
 ######################################
 
+#
+# OUTROTIMER1
+#
 OBJECT=OUTROTIMER1
 OUTROTIMER1:TYPE=TIMER
 OUTROTIMER1:ELAPSE=6000
 OUTROTIMER1:TICKS=0
-OUTROTIMER1:ONTICK={KRAB^PLAY("PLAY");}
-
+OUTROTIMER1:ONTICK=KRABIKPLAY
```

Comment:
This is the only place where this subroutine is used. I'd rather leave it be inline.